states.go

```diff
@@ -219,6 +219,9 @@ type timeSecs struct {
 }
 
 func (t *timeSecs) UnmarshalJSON(b []byte) error {
+	if string(b) == "null" {
```

Comment:
Not sure about these. Might as well make them pointers to time if they're optional?

Reply:
Did we ever get a consensus on this?

Reply:
Current proposal will always have the struct field as zero time instead of nil pointer. Probably more „compatible“ with jsgoecke.